### ISSUE TITLE
Remove Islandora Defaults

### ISSUE THREAD

**Issue by rosiel** [Born-Digital-US/islandora_install_profile_demo#28]
We are soon going to deprecate Islandora Defaults. Islandora Install Profile Demo requires Islandora Defaults. 

Sites using the Install Profile probably want a way to upgrade to a version of the Install Profile that doesn't require Defaults.

**Comment by noahwsmith** [comment]
Good point. "Deprecate" means that the module isn't going away, but it will in a future release, right? So that means we have some time to accomplish this, and meanwhile the module will remain frozen in its current state?

**Comment by rosiel** [comment]
You will still be able to git clone it and the code will not suddenly disappear from your site. However you may soon see warnings in Composer and Drupal that it has been abandoned/deprecated.

More usefully: If you want to use the Mirador module, which has been separated out and is about to get some amazing improvements (including the ability to use plugins like text overlay) then you cannot simultaneously have Islandora Defaults installed.
